test/server/admin/logs_handler_test.cc

```diff
@@ -70,5 +71,90 @@ TEST_P(AdminInstanceTest, LogLevelSetting) {
   EXPECT_EQ(Http::Code::OK, postCallback("/logging?level=warning&paths=", header_map, response));
 }
 
+TEST_P(AdminInstanceTest, LogLevelFineGrainGlobSupport) {
+  Http::TestResponseHeaderMapImpl header_map;
+  Buffer::OwnedImpl response;
+
+  // now for Envoy, w/o setting the mode
```

Comment:
nit: can you clarify what comment means?

Reply:
Removed this.